src/routes/webhook/discord/interaction/confess.ts

```diff
@@ -1,11 +1,12 @@
-import { strictEqual } from 'node:assert/strict';
+import strict, { strictEqual } from 'node:assert/strict';
```

Comment:
The default export is actually the `assert` function 👌 
```suggestion
import assert, { strictEqual } from 'node:assert/strict';
```